docs/developer-guide.rst

```diff
@@ -1,31 +1,40 @@
+***********************************
 Developer Guide
----------------
+***********************************
 
 
 Patch process
-+++++++++++++
+#############
 
 You'd like to contribute features or fixes to Cobbler? Great! We'd love to have them.
 
-It is highly recommended that you have a github.com account if you would like to contribute code. Create an account, log in, and then go to github.com/cobbler/cobbler to "fork" the project.
+It is highly recommended that you have a github.com account if you would like to contribute code. Create an account, log
+in, and then go to github.com/cobbler/cobbler to "fork" the project.
 
-Create a new branch named after the feature you are working on. Do the work on your local machine, please make sure your work passes Cobbler's coding standards by using `make qa`. Only then push to your personal Github branch (e.g. github.com/yourname/cobbler).
+Create a new branch named after the feature you are working on. Do the work on your local machine, please make sure your
+work passes Cobbler's coding standards by using ``make qa``. Only then push to your personal Github branch
+(e.g. github.com/yourname/cobbler).
 
-Then use the "submit pull request" feature of Github to request that the official repo pull in your changes. Be sure to include a full description of what your change does in the comments, including what you have tested (and other things that you may have not been able to test well and need help with).
+Then use the "submit pull request" feature of Github to request that the official repo pull in your changes. Be sure to
+include a full description of what your change does in the comments, including what you have tested (and other things
+that you may have not been able to test well and need help with).
 
 If the patch needs more work, we'll let you know in the comments.
 
-Do not mix work on different features in different pull requests/branches if at all possible as this makes it difficult to take only some of the work at one time, and to quickly slurp in some changes why others get hammered out.
-
-Once we merge in your pull request, you can remove the branch from your repo if you like. The AUTHORS file is created automatically when we release.
+Do not mix work on different features in different pull requests/branches if at all possible as this makes it difficult
+to take only some of the work at one time, and to quickly slurp in some changes why others get hammered out.
 
+Once we merge in your pull request, you can remove the branch from your repo if you like. The AUTHORS file is created
+automatically when we release.
```

Comment:
Is it? Where do we have that?

Reply:
No it is generated through `make authors` which is used while building the rpms with make. This is what this is referring to. I think we need maybe a git hook for automating this or a small bot who updates the `AUTHORS` file in git. I am open to any solution as long as we don't have to maintain it manually.